mock/src/test/java/com/tngtech/keycloakmock/impl/helper/RedirectHelperTest.java

```diff
@@ -145,7 +191,8 @@ void oob_redirect_location_is_generated_correctly() {
     String redirectLocation = uut.getRedirectLocation(session, urlConfiguration);
 
     assertThat(redirectLocation)
-        .isEqualTo("file:///oob-dummy?session_state=session123&state=state123&code=session123");
+        // converting to URI and back to String seems to drop the superfluous //, but that's OK
```

Comment:
Is the file protocol a relevant case for redirects? Otherwise the removal of "//" could make a difference ... Probably the parser does ignore the protocol or interprets it as the hostnname?